packages/components/tailwind.config.js

```diff
@@ -76,4 +76,5 @@ module.exports = {
     transitionProperty: ["motion-safe", "motion-reduce"],
   },
   plugins: [],
+  purge: ["./src/**/*.tsx"],
```

Comment:
maybe we should make this
```suggestion
  purge: ["./src/**/*.{js,jsx,ts,tsx}"],
```
to be most safe & so it doesn't need to be overriden in traject

Reply:
Thanks for the suggestion, committed as suggested